server/core/resource.cc

```diff
@@ -1177,7 +1177,10 @@ HttpResponse resource_handle_request(const HttpRequest& request)
     mxs::Semaphore sem;
     ResourceTask task(request);
 
-    worker->post(&task, &sem);
+    if(!worker->post(&task, &sem))
```

Comment:
Looks good, but add space between if and (.

Reply:
Fixed.